src/views/Profile.vue

```diff
@@ -405,3 +421,10 @@ export default class Profile extends Vue {
   margin: auto .5em auto 0;
 }
 </style>
+
+<style lang="scss">
+
+nav.tabs {
+  overflow: visible;
+}
+</style>
```

Comment:
this will be global

Reply:
@roiLeo if I move this to scoped then the tooltips stop being visible and in buefy there is no attribute to give this class

Reply:
@prachi00 I would remove these lines as it hide horizontal scrollbar on mobile

Reply:
@roiLeo yeah removing this hides the tooltip

> <b-tooltip :label="`Nfts collected by ${shortendId}`">

@roiLeo for sold tab, in #1326 it isn't mentioned